frontend/js/helpers/Category.js

```diff
@@ -1 +1,37 @@
 
+const CATEGORY_API_PATH = "categories";
```

Comment:
I forgot to mention, but please revert this file because these codes have already added in development branch.
Don't need to add again in this branch.